### DISCUSSION TITLE
Feature Request: case insensitive for layer name, st_read() Excel files

### DISCUSSION THREAD

**Discussion by frankjar** [duckdb/duckdb#12614] [Ideas]
Why do you want this feature?

Excel can't have two sheets with the same case insensitive name : 'Sheet1' and 'sheet1'.

ST_READ() function of the spatial extension can specify a sheet with the layer parameter but naming the sheet is case insensitive :

```
select * from st_read('test.xlsx',layer='sheet1');
Binder Error: Layer 'sheet1' could not be found in dataset
```
And using 'Sheet1' instead of 'sheet1' works fine :
```
select * from st_read('test.xlsx',layer='Sheet1');
┌─────────┬─────────┐
│  COL1   │  COL2   │
│ varchar │ varchar │
├─────────┼─────────┤
│ A       │ A       │
│ B       │ B       │
└─────────┴─────────┘
```

Thank you for your interest for the feature.

**Comment by Tishj** [comment]
Please move to `duckdb_spatial` (https://github.com/duckdb/duckdb_spatial)

**Reply by frankjar** [reply]
of course :) 
 it's done : https://github.com/duckdb/duckdb_spatial/discussions/350
